src/Frontend/Fragment/Category.hs

```diff
@@ -25,16 +25,16 @@ import qualified Data.Text as ST
 import qualified Lucid
 import qualified Text.Digestive.Form as DF
 import qualified Text.Digestive.Lucid.Html5 as DF
-import qualified Text.Digestive.Types as DF
 
 
 -- | FIXME: 'makeFormSelectCategory', 'formPageSelectCategory' should be a subform.  (related: `grep
 -- subform src/Frontend/Page/Topic.hs`.)
+-- FIXME: Error and non selected category are inseparable cases.
```

Comment:
> and also some work with the JS code.

that's the key here.  and since the error can only arise from internal errors or adversarial behavior, i would like to leave this as is for now.